otterdog/eclipse-basyx.jsonnet

```diff
@@ -222,6 +222,15 @@ orgs.newOrg('eclipse-basyx') {
         },
       ],
     },
+    orgs.newRepo('basyx-python-framework') {
+      allow_merge_commit: true,
+      allow_update_branch: false,
+      delete_branch_on_merge: false,
+      web_commit_signoff_required: false,
+      workflows+: {
+        default_workflow_permissions: "write",
```

Comment:
I would suggest to use "read" for new repos then you can specify in each workflow the exact permissions it will get.

Reply:
Thanks for the suggestion, I'll change it.